tasks/main.yml

```diff
@@ -22,8 +20,8 @@
   file: path=/opt/dnanexus-upload-agent state=directory mode=0755
 
 - name: Download and unzip UA tarball
-  unarchive: 
-    src: https://dnanexus-sdk.s3.amazonaws.com/dnanexus-upload-agent-1.5.31-linux.tar.gz
+  unarchive:
+    src: https://dnanexus-sdk.s3.amazonaws.com/dnanexus-upload-agent-1.5.33-linux.tar.gz
```

Comment:
we usually pin point versions at the top of the file, so its easier to change it. Would it make sense here?

Reply:
I will move it to vars/main.yml (as that's the way how it works for ansible)